src/tests/ftest/util/server_utils.py

```diff
@@ -204,6 +204,8 @@ def prepare(self, storage=True):
         if storage:
             # Prepare server storage
             if self.manager.job.using_nvme or self.manager.job.using_dcpm:
+                self.log.info("Preparing storage in <format> mode")
```

Comment:
I think this should be reverted, it doesn't fix the issue where wolf-217 is refusing to format because it already has a fs on /mnt/daos, storage is already prepared (/dev/pmemX block devices already exist). The problem to solve is why the /dev/pmem0 on wolf-217 was not erased.